### PR TITLE
Document store's ingress support

### PR DESCRIPTION
**Pull request by SebastianOsuna**
Which other branches should this be merged with (if any)?

### CODE REVIEW

File: scst-store/deployment_details.md [hunk 1]
```diff
@@ -65,3 +66,14 @@ The store will automatically create a read-only cluster role, which may be bound
 ## <a id='export-cert'></a>Exporting certificates
 
 Supply Chain Security Tools - Store creates [Secret Export](https://github.com/vmware-tanzu/carvel-secretgen-controller/blob/develop/docs/secret-export.md) for exporting certificates to `Supply Chain Security Tools - Scan` to securely post scan results. These certificates are exported to the namespace where `Supply Chain Security Tools - Scan` is installed. 
+
+## <a id='ingress'></a>Ingress support
+
+Supply Chain Security Tools - Store's values file allows you to enabled ingress support and to configure a custom domain name to leverage Contour to provide external access to Supply Chain Security Tools - Store's API.
```
Comment (xtreme-debbie-chen) [inline scst-store/deployment_details.md]: Should be: 
> allows you to **enable** ingress support

File: scst-store/ingress_multi_cluster.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+Example:
+```yml
+ingress_enabled: "true"
+ingress_domain: "example.com"
+```
+
+The Supply Chain Security Tools - Store installation will create an HTTPProxy entry with host routing using the qualified name `metadata-store.<ingress_domain>` (`metadata-store.example.com`). The create route supports HTTPS communication via a self-signed certificate with the same subject Alternative Name.
+
+Contour and DNS setup are not part of the Supply Chain Security Tools - Store installation and access to the Supply Chain Security Tools - Store throught Contour depends on the correct configuration of this two components.
```
Comment (xtreme-debbie-chen) [inline scst-store/ingress_multi_cluster.md]: `throught` -> `through`
`this two components` -> `these two components`

File: scst-store/ingress_multi_cluster.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
+Ingress support
```
Comment (xtreme-debbie-chen) [inline scst-store/ingress_multi_cluster.md:1]: Should there be a link to this page on the [TOC page](https://github.com/SebastianOsuna/docs-tap/blob/main/toc.md)?

**Comment by xtreme-debbie-chen** [comment]
Sorry to do this to you. Should there be a blurb in [this page](https://github.com/pivotal/docs-tap/blob/main/scst-store/using_encryption_and_connection.md) about ingress? Specifically it has references to editing the /etc/hosts file which should be unnecessary with ingress.

**Comment by SebastianOsuna** [comment]
> Sorry to do this to you. Should there be a blurb in [this page](https://github.com/pivotal/docs-tap/blob/main/scst-store/using_encryption_and_connection.md) about ingress? Specifically it has references to editing the /etc/hosts file which should be unnecessary with ingress.

Done :)

File: scst-store/using_encryption_and_connection.md [hunk 4]
```diff
@@ -50,22 +50,32 @@ Where:
 
 - `PATH` is the location where you want to save the CA certificate. This file is used later when you [configure the CLI](cli_configuration.md).
 
-###<a id='editing-etchost'></a>Editing `/etc/hosts`
+### <a id='editing-etchost'></a>DNS resolution
```
Comment (xtreme-debbie-chen) [inline scst-store/using_encryption_and_connection.md]: should probably change the id if we're changing the title/name

File: scst-store/using_encryption_and_connection.md [hunk 5]
```diff
@@ -50,22 +50,32 @@ Where:
```
Comment (ohlookadollar) [inline scst-store/using_encryption_and_connection.md:50]: In the CA Certificate section above this: there should be instruction on how to get the CA certificate if ingress in enabled.
ie:
kubectl get secret ingress-cert -n metadata-store -o json | jq -r '.data."ca.crt"' | base64 -d > PATH

**Review by ohlookadollar — CHANGES_REQUESTED** [review]

File: scst-store/using_encryption_and_connection.md (hunk 5 above)
Comment (SebastianOsuna) [reply scst-store/using_encryption_and_connection.md:50]: I added that line

File: scst-store/using_encryption_and_connection.md (hunk 4 above)
Comment (SebastianOsuna) [reply scst-store/using_encryption_and_connection.md]: done

**Pull request merged**